tests/parm/postxconfig-NT-gefs.txt

```diff
@@ -0,0 +1,8233 @@
+1
+222
+GFSPRS
+0
+ncep_nco
+v2003
+local_tab_yes1
+fcst
+oper
+fcst
+ens_fcst
+hour
+nws_ncep
+gefs
+complex_packing_spatial_diff
+2nd_ord_sptdiff
+fltng_pnt
+lossless
+pos_pert_fcst
+12
```

Comment:
@NickSzapiro-NOAA It seems to me the UPP control files "postxconfig-NT-gefs.txt" and "postxconfig-NT-gefs_FH00.txt" haven't been updated to the new format. I wonder if any grib2 files (inline post results) have been successfully  generated from your new RT.

Reply:
Thank you @WenMeng-NOAA . These postxconfig files are from the provided EP5r2 workflow and stopped working after UPP update in #2326.

Do you know how to reformat these?

As temporary fix, I'm using the gfs postxconfig instead. This choice happens in tests/fv3_conf/cpld_control_run.IN
```
#inline post
if [ $WRITE_DOPOST = .true. ]; then
  cp    ${PATHRT}/parm/post_itag_gfs itag
  cp    ${PATHRT}/parm/postxconfig-NT-gfs.txt postxconfig-NT.txt
  cp    ${PATHRT}/parm/postxconfig-NT-gfs_FH00.txt postxconfig-NT_FH00.txt
  cp    ${PATHRT}/parm/params_grib2_tbl_new params_grib2_tbl_new
  if [[ ${BMIC} == .true. ]]; then
    cp    ${PATHRT}/parm/post_itag_gefs itag
    #copied "gefs" postxconfig files not working afer UFS #2326 
    #cp    ${PATHRT}/parm/postxconfig-NT-gefs.txt postxconfig-NT.txt
    #cp    ${PATHRT}/parm/postxconfig-NT-gefs_FH00.txt postxconfig-NT_FH00.txt
    cp    ${PATHRT}/parm/postxconfig-NT-gfs.txt postxconfig-NT.txt
    cp    ${PATHRT}/parm/postxconfig-NT-gfs_FH00.txt postxconfig-NT_FH00.txt
    cp    ${PATHRT}/parm/params_grib2_tbl_new params_grib2_tbl_new
  else
    cp    ${PATHRT}/parm/post_itag_gfs itag
    cp    ${PATHRT}/parm/postxconfig-NT-gfs.txt postxconfig-NT.txt
    cp    ${PATHRT}/parm/postxconfig-NT-gfs_FH00.txt postxconfig-NT_FH00.txt
    cp    ${PATHRT}/parm/params_grib2_tbl_new params_grib2_tbl_new
  fi
fi
```

Reply:
@NickSzapiro-NOAA @lipan-NOAA  If you provide me the UPP control files in xml format, I can generate the text files in new format for you.

Reply:
@NickSzapiro-NOAA I have regenerated "postxconfig-NT-gefs.txt" with the xml file "postcntrl_gefs.xml" provided by @lipan-NOAA. Please let me know if you pick it up from Hera or other machines.

Reply:
@WenMeng-NOAA Can you make PR to this branch with file changes? If not, happy to bring in from Hera

Reply:
@NickSzapiro-NOAA A PR was just submitted to your branch.

Reply:
An example run directory using the updated postxconfig files is on hera at:
`/scratch1/NCEPDEV/nems/Nick.Szapiro/tasks/updateToEP5/uwm_gefs_upp/tests/run_dir/cpld_control_gefs_intel/`
wgrib2 -v seems reasonable, but please let me know if we can verify contents are ok

Reply:
I re-ran and see aerosol fields (same run_dir). Please let me know if there is anything more to resolve

Reply:
@NickSzapiro-NOAA Your test results look good to me. @lipan-NOAA Can you also validate aerosol fields in grib2 files?